go.mod

```diff
@@ -0,0 +1,10 @@
+module example.com
+
+go 1.17
+
+require (
+	WizelineApi v0.0.0
+	github.com/gorilla/mux v1.8.0
+)
+
+replace WizelineApi v0.0.0 => ./
```

Comment:
What do you mean about "local replacements?" in this scenario how should I say it ? The reason of "WizelineApi" is to import packages on the same directory. Should I use "github.com/ErickRodriguezWize/academy-go-q42021"  instead of "WizelineAPI"   for importing packages ?

Reply:
A local replacement is what you are doing. You are basically assigning an alias to your local `./` directory, and importing it. You shouldn't import this package, as it is actually your own project (the module).